### ISSUE TITLE
Latest Stable version (11.69.42.29) does not match a chromedriver to work with

### ISSUE THREAD

**Issue by leleliasos** [openfin/openfin-cli#52]
So I was using ChromeDriver 2.40 which worked great with OpenFin 10.66.41.18 or Chromium 66.0.3359.181 if you prefer. 

However after upgrading to latest OpenFin stable version ( (11.69.42.29)), I started getting an error denoting version mismatch between ChromeDriver and Chromium:

`Chrome version must be >= 66.0.3359.181`

Fair enough, I thought and downloaded Chromedriver 2.44 which is the closest I can get, still getting an error:
`Chrome version must be >= 69.0.3497.0`

So close but no cigar as OpenFin now runs on Chromium 69.0.3497.128
But that means we reached a dead end completely losing the ability to automate OpenFin as if you get the next available ChromeDriver (so 2.45) you get an error that chrome:
`Chrome version must be between 70 and 73`

In other words, latest stable version of OpenFin is not supported but a suitable ChromeDriver.
As a workaround I reverted back to 10.66.41.18 as I'm using a config file to launch my app.

**Comment by wenjunche** [comment]
Hello 

I just tested the following version of ChromeDriver with version 11.69.42.29 of Runtime and it works:

.\chromedriver.exe --version
ChromeDriver 2.38.552522 (437e6fbedfa8762dec75e2c5b3ddb86763dc9dcb)
